lib/login/view/login_form.dart

```diff
@@ -53,13 +55,18 @@ class LoginForm extends StatelessWidget {
 }
 
 class _EmailInput extends StatelessWidget {
+  final FocusNode nextFocusNode;
+
+  _EmailInput(this.nextFocusNode);
+
   @override
   Widget build(BuildContext context) {
     return BlocBuilder<LoginCubit, LoginState>(
       buildWhen: (previous, current) => previous.email != current.email,
       builder: (context, state) {
         return TextField(
           key: const Key('loginForm_emailInput_textField'),
+          autocorrect: true,
```

Comment:
I'm not sure what this line does. So if this line is not necessary remove this line, else keep it.

Reply:
This enables autocorrection on smartphones while entering the mail address. This leads to the intended behavior that your own typical mails are suggested by the smartphone. The value true is also default, the extra configuration clarifies the (intentional) requirement.